components/dashboard/AccountSwitcher.tsx

```diff
@@ -147,7 +147,7 @@ const Option = ({
   description = description || (
     <FormattedMessage
       id="AccountSwitcher.Description"
-      defaultMessage="{type, select, USER {My workspace} COLLECTIVE {Collective admin} ORGANIZATION {Organization admin} EVENT {Event admin} FUND {Fund admin} PROJECT {Project admin} other {}}"
+      defaultMessage="{type, select, USER {My Dashboard} COLLECTIVE {Collective admin} ORGANIZATION {Organization admin} EVENT {Event admin} FUND {Fund admin} PROJECT {Project admin} other {}}"
```

Comment:
"My Dashboard" is a bit confusing to me. I think "Personal profile" or "Personal account" could be better and more in line with the other labels.
```suggestion
      defaultMessage="{type, select, USER {Personal profile} COLLECTIVE {Collective admin} ORGANIZATION {Organization admin} EVENT {Event admin} FUND {Fund admin} PROJECT {Project admin} other {}}"
```